library/tiqr/tests/Tiqr_OCRAWrapperTest.php

```diff
@@ -92,14 +92,14 @@ public function  testOCRAversionDifferences() {
             $v1_length[strlen($response1)]++;
             $response2 = $ocra2->calculateResponse($secret, $challenge, $session_key);
             $v2_length[strlen($response2)]++;
-            if ( str_pad($response1, 6, "0", STR_PAD_LEFT) != str_pad($response2, 10, "0", STR_PAD_LEFT) ) {
+            if ( (int)$response1 != (int)$response2 ) {
```

Comment:
Warning: be careful when comparing values in PHP. This comparison is quite safe as you are explicitly comparing equality of two integer values. By using the `!==` the comparison becomes more strict and tests for values not being identical. 

Not testing for values being identical might lead to bugs in code at a later point. As PHP thinks `(string) '0' == (int) 0`.